json_formatter copy/write.py

```diff
@@ -0,0 +1,37 @@
+import json
+import time as t
+
+class Writer:
+
+    user = {}
+
+    def __init__(self):
+        Writer.user['portfolio'] = {}
+        Writer.user['account_balance'] = 0.0
+        Writer.user['previous_account_values'] = {}
+
+    def update_account_balance(self, balance):
+        Writer.user['account_balance'] = balance
+
+    def update_previous_account_values(self, value):
+        Writer.user['previous_account_values'][str(round(t.time() * 1000000))] = value
+
+    def add_stock(self, name):
+            Writer.user['portfolio']['stock: ' + name] = {
+                'current_qty' : 0,
+                'transactions' : {}
+            }
+
+    def add_transaction(self, name, price, quantity):
+        Writer.user['portfolio']['stock: ' + name]['transactions'][str(round(t.time() * 1000000)) + name.upper()] = {
```

Comment:
Generally, JSON layouts should not have ":" in them as it sometimes messes up the parser. Given that the stocks are already in the portfolio array, I think it's fine to assume that they're stocks (if we add support for options, futures, etc later, we can write a small script to migrate everything in portfolio into a stocks sub-array).